populate.py

```diff
@@ -1177,35 +199,42 @@
 
 
 def gen_sentence():
-    return " ".join(random.sample(lorems, 50))
+    return fake.text()
 
 
 def gen_name():
-    return random.choice(names)
+    from random import shuffle, seed
```

Comment:
This seems odd, do you need to do all of this for each function call? Is there not a direct method in Faker to generate a random first name?

Reply:
exists but can repeat itself at some point